scripts/tests/chiptest/test_definition.py

```diff
@@ -99,12 +100,20 @@ def Run(self, runner, paths: ApplicationPaths):
                     "Unknown test target - don't know which application to run")
 
             tool_cmd = paths.chip_tool
-            if os.path.exists('/tmp/chip_tool_config.ini'):
-                os.unlink('/tmp/chip_tool_config.ini')
 
-            # Remove server all_clusters_app or tv_app storage, so it will be commissionable again
-            if os.path.exists('/tmp/chip_kvs'):
-                os.unlink('/tmp/chip_kvs')
+            files_to_unlink = [
+                '/tmp/chip_tool_config.ini',
+                '/tmp/chip_tool_config.alpha.ini',
+                '/tmp/chip_tool_config.beta.ini',
+                '/tmp/chip_tool_config.gamma.ini',
+                # Remove server all_clusters_app or tv_app storage, so it will be commissionable again
+                '/tmp/chip_kvs',
+                str(Path.home()) + '/Documents/chip.store',
```

Comment:
should we check `if sys.platform == 'darwin'` for this one?

Reply:
> 

Fair enough. I will then add a check for `/tmp/chip_kvs` as I believe it is `Linux` only ?

Reply:
Done.